Ruka-CFW/rk3128-cfw
# Add PS5000 support

A new board has been added to the buildroot and the tree has
been revamped according to the following hierarchy:
1. rockchip common
2. ruka base
3. specific console
  3.1 powkiddy a12
  3.2 ps5000

Brightness control only applies to Powkiddy A12.
Both consoles support HDMI output (must be connected before
switch the console on).
Both consoles support headphones hotplug.


How to test it:
* Run `make <defconfig>` for one of the consoles.
* Run `make`. A rootfs will be created in `output/images/` for the specific console.
* Run `make <defconfig> for the other console.
* Run `make` and you'll get the rootfs for the second console.
* Compare them if you want, check they have what they should according to the board config, compare them with previous versions.
* Flash the consoles and test all features are properly working (HDMI, heaphones, volume control, brightness). Notice that not all features applies to both consoles (brightness doesn't apply to PS5000), and some of them doen't apply to the HDMI mode (headphones, volume control, brightness control).
